Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -2161,11 +2161,11 @@ def mail_command(args, subject_prefix='', in_reply_to=None, references=None):
     send_mail_result = sent_mail_to_entry('Email sent:', [result], email_to, email_from, cc, bcc, rendering_body,
                                           subject)
     if render_body:
-        html_result = CommandResults(
-            entry_type=EntryType.NOTE,
-            content_format=EntryFormat.HTML,
-            raw_response=html_body,
-        )
+        html_result = {
```

Comment:
@adi88d why revert back to raw json? this has not change implication in observing the command result translation to json.

Reply:
I didnt revert back , just change the entry from CommandResults to html entry

Reply:
@adi88d what was there before was already an html entry. returned via CommandResult. there is no need for a change here.

Reply:
Thanks @kobymeir ! i revert this part